frontend/src/components/viewers/ROCCurve.tsx

```diff
@@ -79,11 +79,15 @@ export interface ROCCurveConfig extends ViewerConfig {
 interface ROCCurveProps {
   configs: ROCCurveConfig[];
   maxDimension?: number;
+  colors?: string[];
+  forceLegend?: boolean;
```

Comment:
Why do we need a `forceLegend` here? Would you like to add a comment to explain what it controls?

Reply:
Yes - I described my reasoning briefly here: https://github.com/kubeflow/pipelines/pull/8073#discussion_r930628356. I also added comments in this commit: https://github.com/kubeflow/pipelines/pull/8073/commits/9dfb33b01283477ce5f9e0be3dbef65a2d3353c7. Is that change reasonable?

Reply:
Sounds good. Thank you for adding the comment!